Automattic/wp-calypso
# Purchases: Refactor cancel purchase page to use new layout

#### Proposed Changes

Restyle Cancel Purchase page, using a new layout with a right sidebar and a list of features for WPCom plan. 
P2: pebzTe-aB-p2

#### Design
![image](https://user-images.githubusercontent.com/5706607/206586127-b60b81cf-1b76-47e6-9d98-a3d5e8855ab5.png)

#### Before
<img width="1632" alt="Screenshot 2022-12-09 at 00 15 42" src="https://user-images.githubusercontent.com/5706607/206587017-7ac55c5a-f719-40d5-8331-97b2b154fa0a.png">
<img width="373" alt="Screenshot 2022-12-09 at 00 15 56" src="https://user-images.githubusercontent.com/5706607/206587029-59c3533d-2f6e-4cac-bb71-5cc6d11666f5.png">

#### After
<img width="1633" alt="Screenshot 2022-12-09 at 00 16 44" src="https://user-images.githubusercontent.com/5706607/206587063-65d87f9e-1196-4977-97ba-fee8bc6f8ef1.png">
<img width="374" alt="Screenshot 2022-12-09 at 00 16 31" src="https://user-images.githubusercontent.com/5706607/206587071-3c9d1e4a-1180-45e6-9e20-8561a366d2f5.png">

#### Specific cases

The following are specific cases, for which the page layout changes, as they make use of the usual CancelPurchase component.

## Domains
<img width="1501" alt="case Domain" src="https://user-images.githubusercontent.com/5706607/206710592-11366fd0-9822-4cf3-8c96-582f615d0ba7.png">

## Jetpack products
<img width="1501" alt="case Jepack" src="https://user-images.githubusercontent.com/5706607/206710539-9743369e-1d4d-41f4-b426-c35e0e151ac3.png">

## One-time purchases
<img width="1501" alt="case One Time Purchase" src="https://user-images.githubusercontent.com/5706607/206710567-ed9f2af8-1b5a-4a3c-a19b-8885f694e805.png">

#### Testing Instructions

1. Visit <Calypso>/me/purchases
2. Select a purchase
3. Cancel the purchase
5. Verify the page layout is changed.
6. Verify the Loading Placeholder is updated with the new design
7. Verify the Feature Set for WPCom Plans are properly shown
8. Verify that the copy for Subscriptions accordingly with the design
9. Verify the cancel functionality works properly
10. Verify the responsive behaviour
11. The following are specific cases (see Specific cases section):
    - Domains
    - Jetpack products
    - One-Time purchases

#### Pre-merge Checklist
